src/components/Popup/Popup.tsx

```diff
@@ -96,56 +109,97 @@ export default class Popup extends UIComponent<Extendable<IPopupProps>, IPopupSt
     as: Portal,
     align: 'start',
     position: 'above',
+    accessibility: PopupBehavior as Accessibility,
```

Comment:
would rather suggest the following more robust approach to achieve the same thing:

```jsx
public static defaultProps: IPopupProps = {
  ...
  accessibility: PopupBehavior
}
```

This will ensure that all the changes in the IPopupProps interface (that is used in many other places) will take effect here as well.